### PR TITLE
add mobile user to sms

### PR DESCRIPTION
**Pull request by llussy**
send username to SendSMS.

### CODE REVIEW

File: loda/user.go [hunk 1]
```diff
@@ -36,19 +36,19 @@ func clearUserMap() {
 }
 
 // GetUserMobile return mobile list for users.
-func GetUserMobile(username []string) []string {
-	mobiles := make([]string, len(username))
+func GetUserMobile(username []string) map[string]string {
 	var i int
```
Comment (oiooj) [inline loda/user.go:40]: clean unused var

**Review by oiooj — CHANGES_REQUESTED** [review]

File: loda/user.go [hunk 2]
```diff
@@ -36,19 +36,19 @@ func clearUserMap() {
 }
 
 // GetUserMobile return mobile list for users.
```
Comment (oiooj) [inline loda/user.go:38]: update the function's comment.

**Review by oiooj — CHANGES_REQUESTED** [review]

**Review by llussy — COMMENTED** [review]
ok

**Review by oiooj — APPROVED** [review]

**Comment by oiooj** [comment]
LGTM, Thanks @llussy

**Pull request merged**